shared/chat/conversation/messages/react-button/emoji-picker/index.tsx

```diff
@@ -8,7 +8,9 @@ import {memoize} from '../../../../../util/memoize'
 
 // defer loading this until we need to, very expensive
 const _getData = memoize(() => {
-  const {categories, emojiIndex, emojiNameMap} = require('./data')
+  const categories: typeof Data.categories = require('./data').categories
+  const emojiIndex: typeof Data.emojiIndex = require('./data').emojiIndex
+  const emojiNameMap: typeof Data.emojiNameMap = require('./data').emojiNameMap
```

Comment:
are there any perf implications of using multiple `require()` calls here, rather than just assigning based on one `require()`?

Reply:
no its cached